ginko/src/dts/analysis.rs

```diff
@@ -389,14 +389,14 @@ mod test {
             "\
 /dts-v1/;
 
-/{ 
-    my_l?abel: some_node {}; 
+/{
+    my_l?abel: some_node {};
     my_label_that_has_more_than_31_characters: other_node {};
     some_other_node {
         another_ill#gal_label: sub_node {};
     };
     illegal_node_name#s {};
-};",
+};", ParserContext::default(),
```

Comment:
I would prefer if these changes were the default, i.e., the `PerserContext::default()` is embedded in the `Code` constructor. For the cases where you actually need a non-default `Code` object, I suggest creating a new function, for example named `Code::with_context(String, ParserContext)`, that one should call.
A `Code` object is (and should) just be a simple tool to test a portion of the project without duplicating too much code. This comment is valid for any occurrence of the `Code::new(String, ParserContext)` pattern.